lib/rules/lint-no-trailing-spaces.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const Rule = require('./base');
+
+module.exports = class NoTrailingSpaces extends Rule {
+  visitor() {
+    return {
+
+      // implementation goes here in exit(): in the exit handler, the rule will not
```

Comment:
This comment seems to be out of line with the actual implementation (which is using `enter` of `Program`, not `exit`).

Reply:
I forgot to put it in `exit`. This simplified the tests too.